### DISCUSSION TITLE
paddleOCR与VisualDL显示问题

### DISCUSSION THREAD

**Discussion by Hold-on-li** [PaddlePaddle/PaddleOCR#12178] [Q&A]
配置文件中的step是iter 请问我想将step换成epochs源代码在哪里可以改？

**Comment by LDOUBLEV** [comment]
epoch 参数：https://github.com/PaddlePaddle/PaddleOCR/blob/c9d32d29785ec74dd49fda4a9bf0ec6e0480c903/configs/det/det_mv3_db.yml#L3

**Comment by Hold-on-li** [comment]
> epoch 参数：
> 
> https://github.com/PaddlePaddle/PaddleOCR/blob/c9d32d29785ec74dd49fda4a9bf0ec6e0480c903/configs/det/det_mv3_db.yml#L3

输出的vdl文件还是iter 请问如何修改成epochs
